coords/resolver.py

```diff
@@ -85,17 +85,19 @@ def strtofloat(val):
         return float(val)
 
     except ValueError:
-        raise ValueError("Unable to convert '{0}' to a float.".format(val))
+        raise ValueError(f"Unable to convert '{val}' to a float.") from None
```

Comment:
Showing my python ignorance ... is `raise stuff from None` any different than just `raise stuff` ?  If so then okay, if not then a little confusing.

Reply:
You can 

- raise Blah()
- raise Blah() from None
- raise Blah() from some_previous_exception

What the latter do is change what happens if you have code that has caught exceptions: the None case means something like "hey, don't bother reporting the other errors" whereas giving a previous execption means "the traceback will identify as these are related". The following overly-complicated example hopefully helps:

# using "from None"

Note we get a nice clean trace here, and this is what we want here (we are really just replacing the error with a more-informative one).

```
>>> try:
...     try:
...         raise ValueError("x")
...     except ValueError:
...         raise RuntimeError("bloop") from None
... except ValueError:
...     print("pass")
... 
Traceback (most recent call last):
  File "<stdin>", line 5, in <module>
RuntimeError: bloop
```

# using no "from ..."

Here we see both exceptions but they appear "un-related" - i.e. awkward to track down

```
>>> try:
...     try:
...         raise ValueError("x")
...     except ValueError:
...         raise RuntimeError("bloop")
... except ValueError:
...     print("pass")
... 
Traceback (most recent call last):
  File "<stdin>", line 3, in <module>
ValueError: x

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 5, in <module>
RuntimeError: bloop
```

# with "from ..."

It's still wordy, but at least we know there's some relation between the two exceptions (which can be useful when you want to know what the previous exception was, as is the case in some of the "oh, we failed because of a SSL exception, let's trey again but turning off the SSL context checks" type code we have.

```
>>> try:
...     try:
...         raise ValueError("x")
...     except ValueError as ve:
...         raise RuntimeError("bloop") from ve
... except ValueError:
...     print("pass")
... 
Traceback (most recent call last):
  File "<stdin>", line 3, in <module>
ValueError: x

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 5, in <module>
RuntimeError: bloop
```

HTH